src/components/notifications/noti-buttons.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, h, Prop } from "@stencil/core";
+
+@Component({
+  tag: "oui-noti-buttons",
+  styleUrl: "noti-buttons.css"
+})
+export class NotiButtons {
+  render() {
+    const openTray = () => {
+      const notiTray = document.querySelector("oui-noti-tray");
+      console.log(notiTray);
+      notiTray.open();
+    };
+
+    return (
+      <Host>
+        <button id="buttonUnused">
+          <oui-svg name="noti-bell-unused" scale={0.25}></oui-svg>
+        </button>
+
+        <button id="buttonRead">
+          <oui-svg name="noti-bell-read" scale={0.25}></oui-svg>
```

Comment:
Why name? Shouldn't be class? Name is usually used for form elements.

Reply:
For some reason that was the only way I could get it to reference properly with the oui-svg component.